.github/workflows/sharness.yml

```diff
@@ -52,8 +52,9 @@ jobs:
           TEST_EXPENSIVE: 1
           IPFS_CHECK_RCMGR_DEFAULTS: 1
           CONTINUE_ON_S_FAILURE: 1
+          LIBP2P_TCP_MUX: true # TODO
           # increasing parallelism beyond 10 doesn't speed up the tests much
-          PARALLEL: ${{ github.repository == 'ipfs/kubo' && 10 || 3 }}
+          PARALLEL: 1 # TODO ${{ github.repository == 'ipfs/kubo' && 10 || 3 }}
```

Comment:
This will run test sequentially, hopefully showing us the same `Error: lock [..]kubo/test/sharness/trash directory.t0182-circuit-relay.sh/.iptb/testbeds/default/0/repo.lock: someone else has the lock` as I've hit locally.

Log: https://github.com/ipfs/kubo/actions/runs/12433829774/job/34716226970